homeassistant/components/blink/strings.json

```
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Sign-in with Blink account",
+        "data": { "username": "Username", "password": "Password"}
```

Please use references.
```suggestion
        "data": {
          "username": "[%key:common::config_flow::data::username%]",
          "[%key:common::config_flow::data::password%]": "Password"
        }
```